polaris-react/src/components/CustomProperties/styles.ts

```diff
@@ -7,10 +7,10 @@ import {
   OSColorSchemes,
 } from '../../tokens';
 
-const defaultDeclarations = `
-  ${getColorSchemeDeclarations('light', tokens, osColorSchemes)}
-  ${getStaticCustomProperties(tokens)}
-`;
+const tokenValues = getStaticCustomProperties(tokens);
+const declaration = getColorSchemeDeclarations('light', tokens, osColorSchemes);
+
+const defaultDeclarations = `${declaration}${tokenValues}`;
```

Comment:
```suggestion
const staticCustomProperties = getStaticCustomProperties(tokens);
const colorSchemeDeclarations = getColorSchemeDeclarations('light', tokens, osColorSchemes);

const defaultDeclarations = `${colorSchemeDeclarations}${staticCustomProperties}`;
```

**nit (non-blocking):** I to try keep the same naming convention as the getters unless there is a naming conflict.